.github/actions/test-network-setup/action.yaml

```diff
@@ -3,7 +3,7 @@ description: Set up the Test Network Runtime
 inputs:
   go-version:
     description: Version of go
-    default: '1.22'
```

Comment:
I think the Go version was intentionally moved to 1.22 since 1.21 is no longer supported since Go 1.23 was released. See the [Go release policy](https://go.dev/doc/devel/release#policy).

Reply:
good point, how about we upgrade go mod files into golang 1.22?